### PR TITLE
[IMP] mail: turn 'MessageView/isSelected' into field

### PR DESCRIPTION
**Pull request by alexkuhn**
Task-2793280

### CODE REVIEW

**Comment by oomsveta** [comment]
@alexkuhn conflicting files

**Comment by alexkuhn** [comment]
> @alexkuhn conflicting files

Changes done

File: addons/mail/static/src/components/message/message.xml [hunk 1]
```diff
@@ -13,7 +13,7 @@
                     'o-mobile': messaging.device.isMobile,
                     'o-not-discussion': !(messageView.message.is_discussion or messageView.message.is_notification),
                     'o-notification': messageView.message.message_type === 'notification',
-                    'o-selected': isSelected,
+                    'o-selected':messageView.isSelected,
```
Comment (oomsveta) [inline addons/mail/static/src/components/message/message.xml:16]: missing space

File: addons/mail/static/src/models/message_view.js [hunk 2]
```diff
@@ -218,6 +228,13 @@ registerModel({
          * be set through @see highlight()
          */
         isHighlighted: attr(),
+        /**
+         * Tell whether the message is selected in the current thread viewer.
```
Comment (oomsveta) [inline addons/mail/static/src/models/message_view.js:232]: Tell -> Tells

File: addons/mail/static/src/models/message_view.js [hunk 3]
```diff
@@ -156,6 +156,16 @@ registerModel({
                 this.message.parentMessage
             ) ? insertAndReplace() : clear();
         },
+        /**
+         * @private
+         * @returns {boolean}
+         */
+        _computeIsSelected() {
```
Comment (oomsveta) [inline addons/mail/static/src/models/message_view.js:163]: to be reordered alphabetically

**Comment by alexkuhn** [comment]
@oomsveta changes done

**Comment by oomsveta** [comment]
@alexkuhn conflicting files + runbot angry

**Comment by alexkuhn** [comment]
> @alexkuhn conflicting files + runbot angry

* angry dev noise *

**Comment by alexkuhn** [comment]
Changes done

**Review by phenix-factory — APPROVED** [review]
LGTM

**Comment by alexkuhn** [comment]
@robodoo r+

**Pull request closed without merging**